<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookDialog</class>
 <widget class="QDialog" name="AddBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un livre</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/ressources/img/icons/add_5127035.png</normaloff>:/ressources/img/icons/add_5127035.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="titreLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="categoriesLayout_2">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="agLayout_2">
        <item>
         <widget class="QLabel" name="authorsLabel">
          <property name="text">
           <string>Auteur(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="authorLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="genderLabel">
          <property name="text">
           <string>Genre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="genderLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ecLayout_2">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="editorLabel">
          <property name="text">
           <string>Editeur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editorLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="collectionLabel_2">
          <property name="text">
           <string>Collection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="collectionLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="summarylabel">
      <property name="text">
       <string>Résumé</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="summaryPlainTextEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="neaptcLayout_2">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="neapLayout_2">
        <item>
         <widget class="QLabel" name="exemplaryLabel">
          <property name="text">
           <string>Nb Exemplaires</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="exemplaryLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="yearLabel">
          <property name="text">
           <string>Année de parution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="yearLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="tomecodeLayout_2">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="tomeLayout_2">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="debutTLayout_2">
            <item>
             <widget class="QLabel" name="tomeLabel">
              <property name="text">
               <string>Tome</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="tomeSpinBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="tomeOfLabel">
              <property name="text">
               <string>Sur</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="tomeOfSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="isbnLabel">
          <property name="text">
           <string>Code ISBN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="isbnLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="imageLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLineEdit" name="imgPathEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addImgButton">
        <property name="text">
         <string>Choisir Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBookDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBookDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
